app/_src/policies/meshhttproute.md

```diff
@@ -450,13 +500,13 @@ spec:
                     tags:
```

Comment:
Does the mirror not need to be updated?

Reply:
Unfortunately not yet https://github.com/kumahq/kuma/issues/11341